extensions/wikia/PageShare/PageShareHelper.class.php

```diff
@@ -17,6 +20,29 @@ public static function getIcon( $service ) {
 		return $icon;
 	}
 
+	/**
+	 * Get language for Page Share service.
+	 * For anon users use the browser language, if header is empty default to English.
+	 * For logged in user use user's language.
+	 * Both values can be overwritten by ?uselang parameter.
+	 *
+	 * @return String language
+	 */
+	public static function getLangForPageShare() {
+		global $wgLang, $wgRequest, $wgUser;
+
+		if ( $wgUser->isAnon() ) {
+			if ( array_key_exists( 'HTTP_ACCEPT_LANGUAGE', $_SERVER ) ) {
```

Comment:
How about `isset`?

http://stackoverflow.com/questions/3210935/difference-between-isset-and-array-key-exists
>array_key_exists will definitely tell you if a key exists in an array, whereas isset will only return true if the key/variable exists and is not null